### PR TITLE
修复窗口拖动及优先使用zsh

### PR DESCRIPTION
**Pull request by wenbingzhang**
修复连接标签附近无法拖动窗口问题，及优先使用zsh（主要用于mac os）

### CODE REVIEW

**Comment by zxdong262** [comment]
Hi, @wenbingzhang, Please fix only one bug or add only one feature in one PR.

**Comment by wenbingzhang** [comment]
> Hi, @wenbingzhang, Please fix only one bug or add only one feature in one PR.

1、zsh 只是优先使用,默认还是bash
2、窗口拖动不会影响tab标签的拖动（交换顺序）

**Comment by wenbingzhang** [comment]
> > Hi, @wenbingzhang, Please fix only one bug or add only one feature in one PR.
> 
Hi, @zxdong262 
> 1、zsh 只是优先使用,默认还是bash
> 2、窗口拖动不会影响tab标签的拖动（交换顺序）

**Comment by zxdong262** [comment]
1. No, user do not have a choice to use bash when zsh exists.
2. In mac, it works well, in ubuntu 18.04, seems have problem, I will check again tonight.

**Comment by wenbingzhang** [comment]
> 1. No, user do not have a choice to use bash when zsh exists.
> 2. In mac, it works well, in ubuntu 18.04, seems have problem, I will check again tonight.

嗯，因为我这边没法测试linux和windows，所以我只能确定mac上是正常的。或者我们可以在设置中配置shell

**Comment by zxdong262** [comment]
I checked, add new tab button need `no-drag` in ubuntu to work right.

```stylus
.tabs-add-btn
  display inline-block
  margin 0 3px
  -webkit-app-region no-drag
```
Can you make this in a separate PR?
I really want to merge your PR.

**Comment by wenbingzhang** [comment]
> I checked, add new tab button need `no-drag` in ubuntu to work right.
> 
> ```stylus
> .tabs-add-btn
>   display inline-block
>   margin 0 3px
>   -webkit-app-region no-drag
> ```
> 
> Can you make this in a separate PR?
> I really want to merge your PR.

我尝试一下

File: src/client/components/tabs/tabs.styl [hunk 1]
```diff
@@ -21,7 +21,7 @@
   overflow-y hidden
   margin-left 42px
   -webkit-app-region drag
-.tabs-wrapper
+.tabs-wrapper .tab
   -webkit-app-region no-drag
 .tab
```
Comment (zxdong262) [inline src/client/components/tabs/tabs.styl]: Put a under .tab rules, and remove other files change

Comment (zxdong262) [reply src/client/components/tabs/tabs.styl]: You may put other changes in other PRs

**Comment by zxdong262** [comment]
@wenbingzhang 只保留tabs.styl修改的部分，其他的文件修改在另外的PR中处理，在每个PR中只处理一个问题。

**Pull request closed without merging**